src/client/components/Navigation/NavGeo/RecipeSelector/RecipeSelector.scss

```diff
@@ -0,0 +1,14 @@
+@import 'src/client/style/partials';
+
+.geo-recipe-selector {
+  background-color: $ui-bg-hover;
```

Comment:
This is how it would look like with white instead:

https://github.com/openforis/fra-platform/assets/41337901/f1e90b4d-a41f-4302-9de7-982b645e0b65